modules/table/src/main/java/org/apache/ignite/internal/table/distributed/raft/PartitionListener.java

```diff
@@ -544,23 +550,24 @@ public void onConfigurationCommitted(
                 storage.committedGroupConfiguration(config);
                 storage.lastApplied(lastAppliedIndex, lastAppliedTerm);
 
-                if (!enabledColocation()) {
-                    updateTrackerIgnoringTrackerClosedException(storageIndexTracker, config.index());
-                }
-
                 return null;
             });
 
-            byte[] configBytes = VersionedSerialization.toBytes(config, RaftGroupConfigurationSerializer.INSTANCE);
+            if (!enabledColocation()) {
+                updateTrackerIgnoringTrackerClosedException(storageIndexTracker, config.index());
+
+                byte[] configBytes = VersionedSerialization.toBytes(config, RaftGroupConfigurationSerializer.INSTANCE);
```

Comment:
Do we do this update in the per-zone Raft listener?

Reply:
yes, see `ZonePartitionRaftListener#onConfigurationCommitted`